cmd/monaco/download/download_command_test.go

```diff
@@ -711,59 +826,27 @@ func TestValidCommands(t *testing.T) {
 			test.setup(commandMock)
 
 			cmd := GetDownloadCommand(afero.NewOsFs(), commandMock)
-			cmd.SetArgs(strings.Split(test.args, " "))
+			cmd.SetArgs(test.args)
+			for _, fl := range test.flags {
+				flag := strings.Split(fl, "=")
+				_ = cmd.Flags().Set(flag[0], flag[1])
+			}
 			cmd.SetOut(io.Discard) // skip output to ensure that the error message contains the error, not the help message
 			err := cmd.Execute()
 
-			assert.NilError(t, err, "no error expected")
+			assert.NoError(t, err, "no error expected")
 		})
 	}
 }
 
-// TestInvalidCliCommands is a very basic test testing that invalid commands error.
-// It is not the goal to test the exact message that cobra generates, except if we supply the message.
-// Otherwise, we would run into issues upon upgrading.
-// On th other hand, we could use the exact message to review the exact messages customers will see.
-func TestDisabledCommands(t *testing.T) {
-	t.Setenv("MONACO_FEAT_ENTITIES", "")
-
-	tests := []struct {
-		name                  string
-		args                  string
-		errorContainsExpected []string
-	}{
-		{
-			"entities but is env. var is disabled",
-			"entities",
-			[]string{"unknown command \"entities\" for \"download\""},
-		},
-		{
-			"entities manifest download with project but is env. var is disabled",
-			"entities manifest test.yaml test_env --project testproject",
-			[]string{"unknown command \"entities\" for \"download\""},
-		},
-	}
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			commandMock := createDownloadCommandMock(t)
-
-			cmd := GetDownloadCommand(afero.NewOsFs(), commandMock)
-			cmd.SetArgs(strings.Split(test.args, " "))
-			cmd.SetOut(io.Discard) // skip output to ensure that the error message contains the error, not the help message
-			err := cmd.Execute()
-
-			// for all test cases there should be at least an error
-			assert.Assert(t, err != nil, "there should be an error")
+func TestEntitiesDownloadEnabled(t *testing.T) {
```

Comment:
If I understand the code correctly, the test name is wrong.

This tests a case when Entities download is NOT enabled, verifying that it can't  actually be used.

```suggestion
func TestEntitiesDownloadUnavailableByDefault(t *testing.T) {
```

or something like that

Reply:
Anything related with endities was ignored and left as it was :))

I would to divide entity related code to separate files